pyspecProcScripts/load_data.py

```diff
@@ -38,8 +38,9 @@ def proc_bruker_deut_IR_withecho_mancyc(s, fl=None):
         fl.next("IR prior to FTing ph")
         fl.image(s.C.setaxis("indirect", "#").set_units("indirect", "scan #"))
     s.ft(
-        ["ph1", "ph2"], unitary=True
+        ["ph1", "ph2"]
     )  # fourier transforming from phase cycle dim to coherence dimension
+    s.set_prop("coherence_pathway", {"ph1": 0, "ph2": -1})
```

Comment:
The name implies that it has an echo, so wouldn't Δp₂=+1?  Maybe just leave this out if we're not sure right now.

Reply:
unsure so left a info(strm